infrastructure/nginx/sites-available/development

```diff
@@ -0,0 +1,90 @@
+server {
+	listen 80;
+	listen [::]:80;
+
+	server_name dev.riskopendata.org dev.riskopendata.eu;
+
+	root /var/www/html-dev;
+	index index.html index.htm;
+
+	access_log  /var/log/nginx/access-dev.log combined;
+	error_log   /var/log/nginx/error-dev.log  error;
+
+	keepalive_timeout 20s;
+	sendfile  on;
+	tcp_nopush  on;
+
+	# Compression
+	gzip on;
+	gzip_comp_level 5;
+	gzip_min_length 256;
+	gzip_proxied any;
+	gzip_vary  on;
+	gzip_types
+    application/atom+xml
+    application/javascript
+    application/json
+    application/ld+json
+    application/manifest+json
+    application/rss+xml
+    application/vnd.geo+json
+    application/vnd.ms-fontobject
+    application/x-font-ttf
+    application/x-web-app-manifest+json
+    application/xhtml+xml
+    application/xml
+    font/opentype
+    image/bmp
+    image/svg+xml
+    image/x-icon
+    text/cache-manifest
+    text/css
+    text/plain
+    text/vcard
+    text/vnd.rim.location.xloc
+    text/vtt
+    text/x-component
+		text/x-cross-domain-policy;
+
+	charset utf-8;
+	charset_types
+    text/css
+    text/plain
+    text/vnd.wap.wml
+    application/javascript
+    application/json
+    application/rss+xml
+		application/xml;
+
+	error_page 404 error_404.html
+
+	# Security
+	add_header X-XSS-Protection "1; mode=block" always;
+	add_header X-Content-Type-Options nosniff always;
+	add_header X-Frame-Options SAMEORIGIN always;
+
+	# Cross Domain API
+	add_header "Access-Control-Allow-Origin" "*";
```

Comment:
Are you sure you want to enable CORS to allow access from any host? The API is under `/api` (same domain) so it is not affected by CORS (unless there's a scheme mismatch between client and API requests)